casper/src/test/scala/coop/rchain/casper/api/BlocksResponseAPITest.scala

```diff
@@ -48,7 +49,7 @@ class BlocksResponseAPITest
     for {
       genesis <- createGenesis[Task](bonds = bonds)
       _       <- dagstore.insert(genesis, invalid = false, approved = true)
-      _       <- blockstore.put(genesis)
+      _       <- blockstore.put(genesis.blockHash, genesis)
```

Comment:
Note to use put with one argument here also.

Reply:
[Done](https://github.com/rchain/rchain/pull/3655/commits/7072a5297e0ed0d6872f9ca130d313295078a0ec).